bot/controllers/welcome.js

```diff
@@ -0,0 +1,24 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
```

Comment:
Add in the default comments found across the controllers to your welcome.js file.